.github/workflows/wheels.yml

```diff
@@ -20,8 +20,40 @@ jobs:
         with:
           python-version: '3.10'
       - uses: dtolnay/rust-toolchain@stable
+        with:
+          components: llvm-tools-preview
       - name: Build wheels
         uses: pypa/cibuildwheel@v2.16.2
+        env:
+          CIBW_BEFORE_BUILD: 'bash ./tools/build_pgo.sh'
```

Comment:
I think it'd be nice to accept the merged output location (`/tmp/pgo-data/merged.profdata`) as a parameter to this script so it isn't hard-coded in two separate files, especially if `build_pgo.sh` isn't something developers are calling directly.

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/11502/commits/f3ddf276ba260f546369110733f0e4509d56348c